src/addons/toolbars/GroupedColumnButton.js

```diff
@@ -8,12 +8,19 @@ export default class GroupedColumnButton extends Component {
       textOverflow: 'ellipsis',
       whiteSpace: 'nowrap'
     };
-    return (<button  className="btn grouped-col-btn btn-sm"><span style={style}>{this.props.name}</span>
-    <span className="glyphicon glyphicon-trash" style={{float: 'right', paddingLeft: '5px'}} onClick={this.props.onColumnGroupDeleted.bind(this, this.props.name)}></span></button>);
+    return (
+      <button  className="btn grouped-col-btn btn-sm"><span style={style}>{this.props.name}</span>
+        <span
+          className="glyphicon glyphicon-trash"
+          style={{float: 'right', paddingLeft: '5px'}}
+          onClick={this.props.onColumnGroupDeleted.bind(null, this.props.name)}>
```

Comment:
Line 16: This is the only change of consequence. The rest is just styling as the line was getting too long.

For a discussion on this fix: https://groups.google.com/d/msg/reactjs/Xv9_kVoJJOw/8agybgNhUN4J